### DISCUSSION TITLE
Organizations

### DISCUSSION THREAD

**Discussion by ThatGravyBoat** [modrinth/.github#18] [Feature requests]
### Is your feature request related to a problem? Please describe.

It's hard to organize projects into groups such as a team or organization posting all their mods on the site it's hard for there to be 1 link to show them all without linking to a specific user who may not be on every project or having to create a whole separate account to have this ability.

### Describe the solution you'd like

It would be nice if there were a way for a subset of users to be part of an organization or team that can be used as the owner instead of a single user. 
It would be similar to how discord has teams where when you create an app it asks you if you want it under a group or the user the same thing would happen if you are part of a team on Modrinth and you have the ability to create a repo for that group, when you create a project it can be either part of a user or a team and then that team can have members that have different privileges, it would also have its own "profile" or page where it's like a user profile but it's for the team it can also possibly show the members like how GitHub organizations work if the user lets their organization status be public and most importantly would show all mods that the group is the owner of.

### Describe alternatives you've considered

Customizable lists like the following projects but public, similar to star lists on Github where I can send my star list for others to see would also solve a big part of the problem that teams would fix.

### Additional context

The references to other applications I talked about in the solution and alternatives above.
<details>
  <summary>Discord Dev App Screen: </summary>
  <img src="https://user-images.githubusercontent.com/26934102/205779610-1f25ba99-afac-4c58-bb52-766877ef723c.png"/>
</details>
<details>
  <summary>Github Star List: </summary>
  <img src="https://user-images.githubusercontent.com/26934102/205779767-9fa7e071-8718-48ac-ba4b-41857769a69c.png"/>
</details>

**Comment by triphora** [comment]
Closing since this has been implemented.
